stabilizer/src/main/java/com/hazelcast/stabilizer/test/utils/TestUtils.java

```diff
@@ -114,6 +115,15 @@ public static void assertTrueEventually(AssertTask task) {
         assertTrueEventually(task, ASSERT_TRUE_EVENTUALLY_TIMEOUT);
     }
 
+    public static void assertEqualsByteArray(byte[] expected,byte[] current){
+        if(!(expected.length == current.length)){
```

Comment:
I think you should make use of an assert,not throw an exception.

assertEquals("array lengths not the same", expected.length, current.length)

But does junit not have functionality for array comparisons??